src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/UpdateOrgContactInformationIntegrationTest.java

```diff
@@ -0,0 +1,284 @@
+            .retrieveSingleOrganisation(orgIdentifier, hmctsAdmin);
+        List existingContactsUpdated = (List)retrieveOrganisationResponseAfterUpdate.get("contactInformation");
+        LinkedHashMap existingUpdated = (LinkedHashMap)existingContactsUpdated.get(0);
+        List dxAddressesUpdated = (List)existingUpdated.get("dxAddress");
+        LinkedHashMap dxAdd = (LinkedHashMap)dxAddressesUpdated.get(1);
+        String dxNumber = (String)dxAdd.get("dxNumber").toString();
+        String dxExchange = (String)dxAdd.get("dxExchange").toString();
+
+        assertThat(updateResponse).containsEntry("http_status", 200);
+        assertThat(dxNumber).isEqualTo(contactInformationCreationRequest
+            .getDxAddress().get(0).getDxNumber());
+        assertThat(dxExchange).isEqualTo(contactInformationCreationRequest
+            .getDxAddress().get(0).getDxExchange());
+        deleteOrganisation(orgIdentifier);
+    }
+
+    @Test
+    void shouldReturn400ForNonExistingOrganisation() {
+        ContactInformationCreationRequest contactInformationCreationRequest =
+            createContactInformationRequest().build();
+
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(contactInformationCreationRequest,  hmctsAdmin,
+                "ABCDEF7",true,true,"");
+
+        assertThat(updateResponse).containsEntry("http_status", "404");
+        assertThat(updateResponse.get("response_body").toString())
+            .contains("errorMessage\":\"4 : Resource not found\",\"errorDescription\":\"Organisation does not exist");
+
+    }
+
+    @Test
+    void shoudlReturn400ForInvalidContactInformation() {
+        String organisationIdentifier = createOrganisationRequest();
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(new ContactInformationCreationRequest(null,null,
+                null,null,null,null,null,
+                null,null), hmctsAdmin,organisationIdentifier,true,
+                true,"");
+
+        assertThat(updateResponse).containsEntry("http_status", "400");
+        assertThat(updateResponse.get("response_body").toString())
+            .contains("validation on an argument failed");
+        deleteOrganisation(organisationIdentifier);
+    }
+
+    @Test
+    void shoudlReturn400ForMissingDxAddress() {
+        ContactInformationCreationRequest contactInformationCreationRequest =
+            createContactInformationRequestWithoutDxAddress().build();
+        String organisationIdentifier = createOrganisationRequest();
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(contactInformationCreationRequest, hmctsAdmin,organisationIdentifier,
+                true,true,"");
+
+        assertThat(updateResponse).containsEntry("http_status", "500");
```

Comment:
your test method says should return 400 but you are asserting 500, api should return 400 with error "DX Address details are missing"